src/services/mpEthPriceService.ts

```diff
@@ -0,0 +1,37 @@
+import { logger } from '../utils/logger';
+import Axios, { AxiosResponse } from 'axios';
+import axiosRetry from 'axios-retry';
+
+const mpEthSubgraphUrl = process.env.MPETH_GRAPHQL_PRICES_URL as string;
+
+// Maximum timeout of axios.
+const axiosTimeout = 20000;
+
+const query = {
+  query: `
+    {
+        tokens(where:{id:"0x819845b60a192167ed1139040b4f8eca31834f27"}) {
+            id
+            name
+            symbol
+            decimals
+            lastPriceUSD
+        }
+    }
+    `,
+};
+
+export const fetchMpEthPrice = async () => {
+  try {
+    const result = await Axios.post(mpEthSubgraphUrl, query, {
+      headers: {
+        'Content-Type': 'application/json',
+      },
+      timeout: axiosTimeout,
+    });
+    return Number(result.data.data.tokens[0].lastPriceUSD);
+  } catch (e) {
+    logger.error('fetching Giv Price fetchGivPrice() err', e);
+    throw e;
+  }
+};
```

Comment:
It would be good if you can write a test case for this function
(Just to make sure that we get price and not throw error, and also the price is greater than zero, it would be enough)